nvflare/fuel/hci/server/login.py

```diff
@@ -202,5 +202,10 @@ def pre_command(self, conn: Connection, args: List[str]):
             conn.set_prop(ConnProps.TOKEN, token)
             return True
         else:
-            conn.append_error("not authenticated - no user")
+            conn.append_string("session_inactive")
```

Comment:
These two appends seem to be reversed.  Should be:
`conn.append_error("session_inactive")
conn.append_string("user not authenticated or ...")
`

Basically, the "error" should be terse and easy for machine to check; whereas "string" is for end users.

If change this, you also need to change the logic in API to check the error properly.